modules/getting-started/pages/glossary.adoc

```diff
@@ -62,7 +62,10 @@ API Manager::
 Part of the Anypoint Platform that allows you to design and manage your APIs. It includes the [API Designer], which allows you to write a [RAML] definition of your API. It also includes tools to monitor usage metrics, apply [policies] and to expose interactive documentation through an [API Portal]
```

Comment:
I'm not sure why all those brackets are there. ? [API Designer] [RAML] etc. Maybe it's intentional.

Reply:
Commented out the sentences starting with "It includes" -- the sentences thereafter apply to the API Manager prior to Nov 2017 and are not relevant now.

Reply:
Ah, okay.